apps/src/JSInterpreter.js

```diff
@@ -434,8 +435,9 @@ JSInterpreter.prototype.executeInterpreter = function (firstStep, runUntilCallba
     var err = safeStepInterpreter(this);
     if (!err) {
       var nodeType = this.interpreter.stateStack[0].node.type;
+      this.atInterstitialNode = INTERSTITIAL_NODES.hasOwnProperty(nodeType);
```

Comment:
Looking at Chrome, I actually believe this behavior is correct (breaking when returning to the `SwitchStatement` after executing the discriminant).  Take the following code in the Chrome debugger:
```javascript
  debugger;
  var a = 5;
● switch (a = 7) {
    //
  }
```
When Chrome hits the breakpoint, the value of `a` is 7.  This matches the new behavior of a `switch` breakpoint location.